src/Missings.jl

```diff
@@ -30,7 +30,7 @@ end
 import Base: ==, !=, <, *, <=, !, +, -, ^, /, &, |, xor
 
 if VERSION >= v"0.7.0-DEV.2762"
-    using Base: ismissing, missing, Missing, MissingException
+    using Base: coalesce, ismissing, missing, Missing, MissingException
```

Comment:
IIRC `coalesce` has been added later, when I removed `Nullable`, so it should be under a different version check.